blockchain.go

```diff
@@ -762,6 +787,45 @@ func (b *Blockchain) GetTransactionResult(ID sdk.Identifier) (*sdk.TransactionRe
 	return &result, nil
 }
 
+// GetAccount returns the account for the given address.
+func (b *Blockchain) GetAccountByIndex(index uint) (*sdk.Account, error) {
+
+	generator := flow.NewAddressGenerator(sdk.ChainID(b.vmCtx.Chain.ChainID()))
+
+	generator.SetIndex(index)
+
+	flowAddress := sdkconvert.SDKAddressToFlow(generator.Address())
+
+	account, err := b.getAccount(flowAddress)
+	if err != nil {
+		return nil, err
+	}
+
+	sdkAccount, err := sdkconvert.FlowAccountToSDK(*account)
+	if err != nil {
+		return nil, err
+	}
+
+	return &sdkAccount, nil
+}
+
+func (b *Blockchain) GetAccountUnsafe(address sdk.Address) (*sdk.Account, error) {
```

Comment:
This seems to be a duplicate of `GetAccount` without the locking. Why is it, and not `GetAccount`, called from the debug session?

Reply:
Emulator locking when executing script, and debugger trying to access accounts when stepping over, it was causing deadlock.

Reply:
Sounds good. I'll add a comment/TODO